Add alt texts (#22068)
mdn/content: files/en-us/mozilla/add-ons/webextensions/browser_actions/index.md

A browser action is a button you can add to the browser toolbar. Users can click the button to interact with your extension.

> ![Play button browser action example](browser-action.png)

Reviewer: ```suggestion
![Play button browser action button with an icon that looks like a dog paw print](browser-action.png)
```

> ![Play button browser action button with an icon that looks like a dog paw print](browser-action.png)

Reviewer: ```suggestion
![Play button browser action in the browser toolbar: it looks like a play button](browser-action.png)
```
